install_static_swift_modules/after/Pods/Target Support Files/CustomModuleMapPod-library-iOS/CustomModuleMapPod-library-iOS.unit.xcconfig

```diff
@@ -1,11 +1,10 @@
-FRAMEWORK_SEARCH_PATHS = $(inherited)  "$(PLATFORM_DIR)/Developer/Library/Frameworks"
+FRAMEWORK_SEARCH_PATHS = $(inherited) "$(PLATFORM_DIR)/Developer/Library/Frameworks"
 GCC_PREPROCESSOR_DEFINITIONS = $(inherited) COCOAPODS=1
 HEADER_SEARCH_PATHS = $(inherited) "${PODS_ROOT}/Headers/Private" "${PODS_ROOT}/Headers/Private/CustomModuleMapPod" "${PODS_ROOT}/Headers/Public"
 LD_RUNPATH_SEARCH_PATHS = $(inherited) '@executable_path/Frameworks' '@loader_path/Frameworks'
 LIBRARY_SEARCH_PATHS = $(inherited) "${PODS_CONFIGURATION_BUILD_DIR}/CustomModuleMapPod-library-iOS"
 OTHER_CFLAGS = $(inherited) -fmodule-map-file="${PODS_ROOT}/Headers/Private/CustomModuleMapPod/CustomModuleMapPod-library-iOS.modulemap"
-OTHER_LDFLAGS = -ObjC -l"CustomModuleMapPod-library-iOS" -framework "XCTest"
-OTHER_SWIFT_FLAGS = $(inherited) -Xcc -fmodule-map-file="${PODS_ROOT}/Headers/Private/CustomModuleMapPod/CustomModuleMapPod-library-iOS.modulemap"
```

Comment:
same question is this supposed to be removed?

Reply:
yes